### ISSUE TITLE
Dataset problem: CESM2-FV2 and CESM2-WACCM-FV2 models exists on disk but not in CMIP6_CV.json

### ISSUE THREAD

**Issue by ledm** [ESMValGroup/ESMValCore#518]
The model data: `CESM2-FV2` and `CESM2-WACCM-FV2` exists on disk on jasmin, but is not listed in `esmvalcore/cmor/tables/cmip6/Tables/CMIP6_CV.json`.

On Jasmin, these data exist here:
```
/badc/cmip6/data/CMIP6/CMIP/NCAR/CESM2-FV2/
/badc/cmip6/data/CMIP6/CMIP/NCAR/CESM2-WACCM-FV2
```

Several other `CESM2` based models are listed though:

- CESM2
- CESM2-SE
- CESM2-WACCM

**Comment by ledm** [comment]
edit: found another model from NCAR not listed.

**Comment by zklaus** [comment]
These have been added to the CV 2019-08-17 as CV version [6.2.35.5](https://github.com/WCRP-CMIP/CMIP6_CVs/tree/6.2.35.5) and [6.2.35.6](https://github.com/WCRP-CMIP/CMIP6_CVs/tree/6.2.35.5). Let me check if and when that found its way into the tables.

**Comment by zklaus** [comment]
They are in the tables since 2019-08-18, but only in the repository, there is apparently no release including them yet.

**Comment by zklaus** [comment]
So what should we do with this? The data is correct, the modelers did everything as they should and as soon as we update our tables, it will work for us as well.

@ledm, does this currently produce a problem?

**Comment by valeriupredoi** [comment]
we are on CMOR tables:
```
"data_specs_version": "01.00.31"
"table_date": "24 July 2019"
```
Should we update them tables to what since the last stable version I see is indeed `01.00.31`?

**Comment by valeriupredoi** [comment]
> They are in the tables since 2019-08-18, but only in the repository, there is apparently no release including them yet.

ahaa, that explains my previous comment - @zklaus I don't feel comfortable to update to a version that's not a release, but you know more about this shtuffs

**Comment by zklaus** [comment]
I tend to agree. I didn't mean to imply that we should upgrade them now, just that when we upgrade them in, I don't know, a years time, things will fall into place.

**Comment by zklaus** [comment]
We might, however, consider getting our CV directly from the source, particularly in light of [the note on the cmip6 tables repository](https://github.com/PCMDI/cmip6-cmor-tables)
> Note:
>
> Only CMOR and PrePARE should use the JSON file of concatenated CMIP6 Control Vocabulary  CMIP6_CV.json found in this repository.
>
> All other software should get the CV's directly from the WCRP repository.

**Comment by valeriupredoi** [comment]
well that's a very useful piece of note - yeah, we need to probably - @jvegasbsc @bouweandela

**Comment by bouweandela** [comment]
I would just wait for the next cmor tables update, they do them pretty regularly. In the meantime, I guess we can add the missing experiments to config-developer.yml, like we have for CMIP5, and that should just work.

**Comment by valeriupredoi** [comment]
So we're slow but sure rather than living on the edge :grin:

**Comment by bouweandela** [comment]
> I guess we can add the missing experiments to config-developer.yml, like we have for CMIP5, and that should just work.

Or specify the `activity` and `institute` in the dataset definition in the recipe, that would probably be even easier.

**Comment by jvegasbsc** [comment]
In fact we are supposed to get CV not from the cmor tables repository (see https://github.com/PCMDI/cmip6-cmor-tables), but from the WCRP-CMIP repo https://github.com/WCRP-CMIP/CMIP6_CVs. 

I will open an issue so I fix it when I have time.

**Comment by zklaus** [comment]
One thing I am wondering about is how the data centres deal with this.

@valeriupredoi, @bjoernbroetz, maybe you could get some info from jasmin and mistral, respectively? In particular, do they move files when things like the institute change? Are such changes forbidden once some data has been published?

@valeriupredoi, have you started a list with topics to take up at the ESGF F2F? This could be something.

**Comment by valeriupredoi** [comment]
> One thing I am wondering about is how the data centres deal with this.
> 
> @valeriupredoi, @bjoernbroetz, maybe you could get some info from jasmin and mistral, respectively? In particular, do they move files when things like the institute change? Are such changes forbidden once some data has been published?

good call, I will investigate on Jasmin/ESGF node :beer:

> @valeriupredoi, have you started a list with topics to take up at the ESGF F2F? This could be something.

No, not 100% sure that will happen given covid19 but then again we should probably do it anyway

One remark: maybe it would be useful to have gitHub contacts from model developers so we can tag them right away in model-specific issues here? That might switch off delays over email and they may respond right away with 'nay, this is something we won't do' or 'yay, we aill log it and work on it'
